build.gradle

```diff
@@ -33,6 +33,9 @@ buildscript {
     // Dagger
     ext.dagger_version = "2.24"
 
+    //Koin
```

Comment:
Optional: Missing a little of space for consistency
Replace -> `//Koin` with `// Koin`

Reply:
Oh yes. 
I'll change this.